core/engine.h

```diff
@@ -95,6 +96,7 @@ class Engine {
 	bool is_in_physics_frame() const { return _in_physics; }
 	uint64_t get_idle_frame_ticks() const { return _frame_ticks; }
 	float get_idle_frame_step() const { return _frame_step; }
+	float get_physics_interpolation_fraction() const {return _physics_interpolation_fraction;}
```

Comment:
Please add spaces before and after the `{}` as seen on the lines above:

```suggestion
	float get_physics_interpolation_fraction() const { return _physics_interpolation_fraction; }
```